Add missing documentation for README.md (#116)
fastify/session: README.md

The options object is used to generate the `Set-Cookie` header of the session cookie. May have the following properties:
* `path` - The `Path` attribute. Defaults to `/` (the root path).
* `maxAge` - A `number` in milliseconds that specifies the `Expires` attribute by adding the specified milliseconds to the current date. If both `expires` and `maxAge` are set, then `maxAge` is used.

> * `rolling` - Forces the session identifier cookie to be set on every response. The expiration is reset to the original maxAge, resetting the expiration countdown. This is typically used in conjuction with short, non-session-length maxAge values to provide a quick timeout of the session data with reduced potential of it occurring during on going server interactions. Defaults to true.

Reviewer: ```suggestion
* `rolling` - Forces the session identifier cookie to be set on every response. The expiration is reset to the original maxAge and resetting the cookie lifetime. This is typically used in conjuction with short, non-session-length maxAge values to provide a quick expiration of the session data with reduced potential of session expiration occurring during ongoing server interactions. Defaults to true.
```

> * `rolling` - Forces the session identifier cookie to be set on every response. The expiration is reset to the original maxAge and resetting the cookie lifetime. This is typically used in conjuction with short, non-session-length maxAge values to provide a quick expiration of the session data with reduced potential of session expiration occurring during ongoing server interactions. Defaults to true.

Reviewer: ```suggestion
* `rolling` - Forces the session identifier cookie to be set on every response. The expiration is reset to the original maxAge - effectively resetting the cookie lifetime. This is typically used in conjuction with short, non-session-length maxAge values to provide a quick expiration of the session data with reduced potential of session expiration occurring during ongoing server interactions. Defaults to true.
```
